qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/pairing_table_widget/pairing_table_constants.py

```diff
@@ -0,0 +1,18 @@
+# pair_table_constants.py
+from typing import List
+
+inverse_pair_columns = {"pair_name": 0, "to_analyse": 1, "group_1": 2, "group_2": 3, "alpha": 4, "guess_alpha": 5}
+
+
+def get_index_of_text(selector, text):
```

Comment:
I have found this function declared exactly the same in other parts of MUON interfaces. This is outside the scope of this issue, but can we create an issue to investigate wheter we can declare this function on some utils file of the utilities folder of the class?

Reply:
Added #38315